### ISSUE TITLE
cloud you please support rgba color

### ISSUE THREAD

**Issue by yzhang921** [odedglas/linear-gradient-parser#3]
```
const linearGradient = {
            x1: '17.5001',
            y1: '32',
            x2: '17.5001',
            y2: '2.9711',
            stops: [
                { color: 'rgba(255, 69, 0, 0.68)', offset: '0'},
                { color: 'rgba(109,245,255,0.68)', offset: '1' }
            ]
        };
 const jsonResult = lgParser.getBackground(linearGradient);
```

**Comment by odeadglaz** [comment]
Will do! thanks :)

**Comment by odeadglaz** [comment]
@yzhang921 , Solved on 1.1.5.
ref: https://github.com/odedglas/linear-gradient-parser/commit/bd3a34d9d03b1d0e69fa9fb6b97504e8bd965103
